src/runtime/disco/nccl/nccl.cc

```diff
@@ -137,20 +137,12 @@ inline ncclRedOp_t AsNCCLRedOp(ReduceKind kind) {
 struct CCLThreadLocalContext {
   DiscoWorker* worker;
   int device_id;
-  deviceStream_t default_stream;
+  deviceStream_t default_stream = nullptr;
   ncclComm_t comm;
 
-  void Clear() {
-    NCCL_CALL(ncclCommDestroy(comm));
-    StreamDestroy(default_stream);
-  }
+  void Clear() { NCCL_CALL(ncclCommDestroy(comm)); }
 
-  deviceStream_t GetDefaultStream() {
-    const auto* func = tvm::runtime::Registry::Get("runtime.get_" TVM_DISCO_DEVICE_NAME "_stream");
-    ICHECK(func != nullptr);
-    deviceStream_t stream = static_cast<deviceStream_t>((*func)().operator void*());
-    return stream == nullptr ? default_stream : stream;
-  }
```

Comment:
it seems to break cudagraph integration because cudagraph provides a default stream